tvtk/pyface/ui/qt4/decorated_scene.py

```diff
@@ -158,6 +158,7 @@ def _save_snapshot(self):
                             "Geomview", "VRML", "Wavefront", "Open Inventor",
                             "Povray", "X3D"]
 
+            wildcard = ""
```

Comment:
Of course, but the method is private too so it smells that I have to invoke `_save_snapshot` but anyway, better a simple test than none.  Testing UI specific stuff always leaves me with a bad taste and I am never sure if it is a good use of one's time.  I wish there were easier tooling to make this easier to do or write the code in a way that we could focus on other things.  For example, there are some scenarios that are painful to test, for example if you click on a traits UI color selector from within IPython with %gui=qt it disappears -- how is one supposed to be able to test this?  For example try this:

```
mlab.test_plot3d()
```

Now click on the last (gear) icon on the toolbar and try changing the background color, the UI appears and disappears.  How should we test this?  Its frustrating that there aren't easier ways to ensure that these do not happen due to changes in traitsui/qt/pyface/whatever.